### ISSUE TITLE
While I am running the program, why Gym-Torch constantly re-launch?!!

### ISSUE THREAD

**Issue by hbzhang** [abhisheknaik96/MultiAgentTORCS#12]
While I am running the program, why Gym-Torch constantly re-launch?!!

**Comment by abhisheknaik96** [comment]
Hello, the gym-torcs relaunches when the episode for any one of the agent ends due to some termination condition (out of track, facing backwards, etc). If you are facing some other issue, please do provide more details. 

Additionally, please check out the [new repo](https://github.com/madras-simulator/MADRaS) for this project!

**Comment by hbzhang** [comment]
Thanks, that means restarting is normal?

I am using single-agent I believe.

Restarting is pretty distracting since screen constantly coming up...

**Comment by abhisheknaik96** [comment]
Yes, restarting is normal! Even in the single-agent case, there will be more restarting towards the beginning of training as the agent will fail frequently. 
Make sure to check out [the Wiki](https://github.com/abhisheknaik96/MultiAgentTORCS/wiki/Hacks-for-DRL-with-TORCS) for the repo for more insights into how this works!
